keps/prod-readiness/sig-apps/3715.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 3715
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@soltysh"
```

Comment:
Please switch back to me - we're generally not changing approvers in the middle.

Reply:
sg, done.